src/jats/importer/parse-jats-article.ts

```diff
@@ -70,6 +71,15 @@ export const parseJATSFront = (doc: Document, front: Element) => {
     ...front.querySelectorAll('article-meta > author-notes > corresp'),
   ])
 
+  const authorNotesParagraphs = jatsFrontParser.parseAuthorNotesParagraphs([
+    ...front.querySelectorAll('article-meta > author-notes > p'),
+  ])
+  const authorNotes = buildAuthorNotes(
+    Array.from(footnoteIDs.values()).concat(
+      authorNotesParagraphs.map((p) => p._id)
+    )
+  )
+
```

Comment:
I think we should combine the p and fn parsing in a single `jatsFrontParser.parseAuthorNotes`.

Reply:
A question about this.. are the author-notes optional or required in the jats? we shouldn't generate a model if we don't have an author-notes element right? Or do we generate it anyways?

Reply:
I believe it's optional, and yes we should not generate a model if it doesn't exist.

Reply:
I updated the code, please let me know if any more changes are needed